nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -339,11 +348,35 @@ def do_compression(
         nncf_logger.info(self._get_bitwidth_distribution_str(all_weight_params, ratio_defining_params))
 
         if self._awq and activations is not None and self._mode != CompressWeightsMode.NF4:
+            awq_params = self._advanced_parameters.awq_params
             awq_algo = AWQ(
-                model, self._backend_entity.name_to_node_mapping, all_weight_params, nodes_to_compress, activations
+                model,
+                self._backend_entity.name_to_node_mapping,
+                all_weight_params,
+                nodes_to_compress,
+                activations,
+                awq_params.subset_size,
+                awq_params.percent_to_apply,
+                awq_params.alpha_min,
+                awq_params.alpha_max,
+                awq_params.steps,
             )
             awq_algo.apply(model, graph)
 
+        if self._scale_estimation and activations is not None and self._mode != CompressWeightsMode.NF4:
+            scale_estimation_params = self._advanced_parameters.scale_estimation_params
+            scale_algo = ScaleEstimation(
```

Comment:
May be, but in this case we have to completely rewrite current pipeline.

Reply:
My opinion that it will not lead to rewriting the pipeline because you will have something like that:
```
if self._scale_estimation and activations is not None and self._mode != CompressWeightsMode.NF4:
    scale_estimation_params = self._advanced_parameters.scale_estimation_params
    scale_algo = ScaleEstimation(
        model,
        self._backend_entity.name_to_node_mapping,
        all_weight_params,
        nodes_to_compress,
        activations,
        scale_estimation_params.subset_size,
        scale_estimation_params.initial_steps,
        scale_estimation_params.scale_steps,
    )
    scale_algo.apply(model, graph)
else:
    # Compress model using weight compression parameters
    transformed_model = self._backend_entity.transform_model(
        model, graph, track(all_weight_params, description="Applying Weight Compression")
    )
```